samples/sample_hevc_fei/src/fei_utils.cpp

```diff
@@ -533,6 +547,13 @@ void FieldSplitter::Close()
     return m_pTarget->Close();
 }
 
+mfxStatus FieldSplitter::ResetIOState()
+{
+    m_pTarget->ResetIOState();
```

Comment:
you should return status of ResetIOState() here

Reply:
Agree. Thanks.